lib/std/fs/path.zig

```diff
@@ -484,6 +484,7 @@ pub fn resolve(allocator: Allocator, paths: []const []const u8) ![]u8 {
 /// Path separators are canonicalized to '\\' and drives are canonicalized to capital letters.
 /// Note: all usage of this function should be audited due to the existence of symlinks.
 /// Without performing actual syscalls, resolving `..` could be incorrect.
+/// assume: paths describes network path (prefix `\\`) or root path (prefix `X:` or `x:`)
```

Comment:
This comment still looks incorrect to me - This function intends to support relative paths, resolving versus the CWD if necessary

Same for `resolvePosix`

Reply:
I will change it then to `TODO: support relative paths`

Reply:
What's the case that's failing? There a couple of basic relative path tests that seem to be passing already

Reply:
added the TODOs.

Reply:
There are already working test cases at https://github.com/ziglang/zig/blob/f59cbd89e349fb3002500cb2a1d69ca5e6063338/lib/std/fs/path.zig#L736-L751

If you're noticing a failure, I think that should be filed as a separate issue